docs/pages/enterprise/workflow/ssh-approval-jira-cloud.mdx

```diff
@@ -82,19 +82,34 @@ In the webhook settings page, make sure that the webhook will only send Issue Up
 
 We recommend installing Teleport plugins alongside the Teleport Proxy. This is an ideal
 location as plugins have a low memory footprint, and will require both public internet access
-and Teleport Auth access. We currently only provide linux-amd64 binaries, you can also
-compile these plugins from [source](https://github.com/gravitational/teleport-plugins/tree/master/access/jira).
+and Teleport Auth access. 
 
+<Tabs>
+<TabItem label="Download">
```

Comment:
The fluentd preexisting `<Tabs>` has two spaces of indentation. :shrug: Not sure if we care about that -- I defer to @ptgott.